### PR TITLE
Flag IndexPathUpdates as having changes when modified items are set

### PR DESCRIPTION
**Pull request by cdzombak**
closes https://github.com/wtmoose/TLIndexPathTools/issues/51

### CODE REVIEW

File: TLIndexPathTools/Data Model/TLIndexPathUpdates.m [hunk 1]
```diff
@@ -403,6 +403,7 @@ - (void)performBatchUpdatesOnCollectionView:(UICollectionView *)collectionView c
 - (void)setModifiedItems:(NSArray *)modifiedItems
 {
     _modifiedItems = [modifiedItems copy];
+    _hasChanges = _modifiedItems.count != 0;
```
Comment (wtmoose) [inline TLIndexPathTools/Data Model/TLIndexPathUpdates.m]: Can you change this to:

    self.hasChanges = self.hasChanges || _modifiedItems.count != 0;

Comment (cdzombak) [inline TLIndexPathTools/Data Model/TLIndexPathUpdates.m]: good catch. silly mistake on my part.

Comment (cdzombak) [inline TLIndexPathTools/Data Model/TLIndexPathUpdates.m]: it's a `readonly`

**Comment by cdzombak** [comment]
thanks, @wtmoose! Awesome library; glad i could help.

**Comment by wtmoose** [comment]
I made a minor updated, changing `hasChanges` to be internally `readwrite`.

Thanks for the fix!

**Pull request merged**